packages/integration-tests/suites/replay/multiple-pages/test.ts

```diff
@@ -45,21 +45,31 @@ sentryTest(
 
     const reqPromise0 = waitForReplayRequest(page, 0);
     const reqPromise1 = waitForReplayRequest(page, 1);
+    const reqPromise2 = waitForReplayRequest(page, 2);
```

Comment:
(no action required)
Good change, let's just always keep these at the top of tests for consistency going forward